modules/serverless-kafka-tls-sasl.adoc

```diff
@@ -0,0 +1,87 @@
+// Module is included in the following assemblies:
+//
+// serverless/serverless-kafka.adoc
+
+[id="serverless-configuring-tls-authentication-against-an-apache-kafka_{context}"]
+= Configuring authentication for Kafka
+
+In production, Kafka clusters are often secured using the TLS or SASL authentication methods. This section shows how to configure the `KafkaChannel` to work against a protected Kafka cluster using TLS or SASL.
+
+== Configuring TLS authentication
+
+.Prerequisites
+
+* A CA certificate as a `.pem` file.
+* A client certificate and key as `.pem` files.
```

Comment:
Not really clear where a user gets these from, is this something they create?
It looks like the first step in the procedure might actually create these, so if it does I don't think they need to be listed as prereqs?

Reply:
If I understand correctly, the user needs to get them in advance -- the first step in the procedure uses these files. Is it correct @matzew ?

Reply:
Correct. user setting up the channel for TLS/SASL needs to get them in advance. I'd guess from the person that is in charge of the used Apache Kafka cluster. (e.g. its admin)

Reply:
here, I replied to the original comment via email a couple of days ago:

https://github.com/openshift/openshift-docs/pull/29059#issuecomment-781536990